### PR TITLE
Update Human Pose Tracking Readme file

### PR DESCRIPTION
**Pull request by andreasnaoum**
### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/5359/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/5359/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/5359/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5359)
- [Docs preview](https://rerun.io/preview/6e40df143bfeb5f81bbee53a8eb2454b02581555/docs) 
- [Examples preview](https://rerun.io/preview/6e40df143bfeb5f81bbee53a8eb2454b02581555/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

**Comment by emilk** [comment]
Rendered: https://github.com/rerun-io/rerun/blob/andreasnaoum-readmes/examples/python/human_pose_tracking/README.md

File: examples/python/human_pose_tracking/README.md
Comment (emilk) [inline examples/python/human_pose_tracking/README.md:80]: I suggest we remove this section, because it will very likely code-rot (if we update the code we'll forget to update the README). Just tell the user to run `python examples/python/gesture_detection/main.py --help` and they can read it themselves

File: examples/python/human_pose_tracking/README.md
Comment (emilk) [inline examples/python/human_pose_tracking/README.md:95]: nit: we prefer to use `…` everywhere for consistency

```suggestion
    # … existing code …
```

**Review by emilk — APPROVED** [review]

**Comment by emilk** [comment]
Hot tip: configure your editor to trim trailing whitespace, and to always add a ending newline.

Then run `pixi run lint-rerun` to see some of the errors that the CI is complaining about

File: examples/python/human_pose_tracking/README.md
Comment (roym899) [inline examples/python/human_pose_tracking/README.md:43]: Any reason for the empty lines for each item?

Comment (roym899) [reply examples/python/human_pose_tracking/README.md:43]: Some inconsistencies:
- Period / no period at end
- Inconsistent tense for verbs

File: examples/python/human_pose_tracking/README.md
Comment (roym899) [inline examples/python/human_pose_tracking/README.md:86]: "Rerun's timeline functions" instead of "Rerun timelines' functions"?

File: examples/python/human_pose_tracking/README.md
Comment (roym899) [inline examples/python/human_pose_tracking/README.md:87]: "As a result," seems a bit off to me. As a result of what? I'd probably write sth like: "Using those, each frame can be logged at a specific timestamp to one or multiple timelines."

File: examples/python/human_pose_tracking/README.md
Comment (roym899) [inline examples/python/human_pose_tracking/README.md:88]: Missing "this" and "Rerun's functions" is out of place. How about: "Here we use Rerun's `set_time_seconds` and `set_time_sequence` functions to set a continuous and discrete timestamp, respectively."

File: examples/python/human_pose_tracking/README.md
Comment (roym899) [inline examples/python/human_pose_tracking/README.md:112]: Skeleton?

File: examples/python/human_pose_tracking/README.md
Comment (roym899) [inline examples/python/human_pose_tracking/README.md:115]: "In both 2D and 3D points, specifying connections between points is essential."
This sentence doesn't make sense to me. "For both ..."; but also its not essential. I'd remove this sentence completely.

File: examples/python/human_pose_tracking/README.md
Comment (roym899) [inline examples/python/human_pose_tracking/README.md:117]: I think it would be more useful if this would say what kind of datastructure is contained in POSE_CONNECTIONS.

File: examples/python/human_pose_tracking/README.md
Comment (roym899) [inline examples/python/human_pose_tracking/README.md:188]: I think `keypoint_ids=mp_pose.PoseLandmark` is the non-obvious part here that would be nice to highlight in the text before.

**Review by roym899 — COMMENTED** [review]
I think the writing should be improved a bit before merging this; I added a few comments.

On a higher level, I wonder if the code snippets should only focus on Rerun's logging function. I think the interesting snippets from Rerun's perspective are:

```python
rr.log(
    "/",
    rr.AnnotationContext(
        rr.ClassDescription(
            info=rr.AnnotationInfo(id=0, label="Person"),
            keypoint_annotations=[rr.AnnotationInfo(id=lm.value, label=lm.name) for lm in mp_pose.PoseLandmark],
            keypoint_connections=mp_pose.POSE_CONNECTIONS,
        )
    ),
    timeless=True,
)
```


```python
rr.set_time_seconds("time", bgr_frame.time)
rr.set_time_sequence("frame_idx", bgr_frame.idx)
```

```python
rr.log("video/rgb", rr.Image(rgb).compress(jpeg_quality=75))
```

```python
rr.log(
    "video/pose/points", 
    rr.Points2D(landmark_positions_2d, class_ids=0, keypoint_ids=mp_pose.PoseLandmark)
)
```
```python
rr.log(
    "person/pose/points",
    rr.Points3D(landmark_positions_3d, class_ids=0, keypoint_ids=mp_pose.PoseLandmark),
)
```

They could all be explained nearly in isolation by saying what datastructure the non-obvious arguments are. I think now there is a lot of code on MediaPipe; however, I'd assume that's not what most people are looking for when looking at an example README (and if they do, that information would still be available in the example code).

**Comment by andreasnaoum** [comment]
I changed completely the readme, so please take a quick look again.

We are trying to figure out the template for README files. 

Suggested Template: 

Small Description
- Used Rerun Types
- Background (Optional)
- Logging and Visualizing with Rerun 
- Run

File: examples/python/human_pose_tracking/README.md
Comment (nikolausWest) [inline examples/python/human_pose_tracking/README.md:18]:
```suggestion
Use the [MediaPipe Pose Landmark Detection](https://developers.google.com/mediapipe/solutions/vision/pose_landmarker) solution to detect and track a human pose in video.

```

File: examples/python/human_pose_tracking/README.md
Comment (nikolausWest) [inline examples/python/human_pose_tracking/README.md:25]:
```suggestion
The [MediaPipe Pose Landmark Detection](https://developers.google.com/mediapipe/solutions/vision/pose_landmarker) solution detects and tracks human pose landmarks and produces segmentation masks for humans. The solution targets real-time inference on video streams. In this example we use Rerun to visualize the output of the Mediapipe solution over time to make it easy to analyze the behavior.
```

File: examples/python/human_pose_tracking/README.md
Comment (nikolausWest) [inline examples/python/human_pose_tracking/README.md:33]:
```suggestion
For each processed video frame, all data sent to Rerun is associated with the two [`timelines`](https://www.rerun.io/docs/concepts/timelines) `time` and `frame_idx`.
```

**Review by nikolausWest — APPROVED** [review]
Ready to merge after suggestions

File: examples/python/human_pose_tracking/README.md
Comment (nikolausWest) [inline examples/python/human_pose_tracking/README.md:86]:
```suggestion
Logging the body pose landmarks involves specifying connections between the points, extracting pose landmark points and logging them to the Rerun SDK.
The 2D points are visualized over the image/video for a better understanding and visualization of the body pose. The 3D points allows the creation of a 3D model of the body posture for a more comprehensive representation of the human pose.
```

File: examples/python/human_pose_tracking/README.md
Comment (nikolausWest) [inline examples/python/human_pose_tracking/README.md:96]:
```suggestion
The 2D and 3D points are logged through a combination of two archetypes. First, a timeless
[`ClassDescription`](https://www.rerun.io/docs/reference/types/datatypes/class_description) is logged, that contains the information which maps keypoint ids to labels and how to connect
the keypoints.
Defining these connections automatically renders lines between them. Mediapipe provides the `POSE_CONNECTIONS` variable which contains the list of `(from, to)` landmark indices that define the connections. Second, the actual keypoint positions are logged in 2D
and 3D as [`Points2D`](https://www.rerun.io/docs/reference/types/archetypes/points2d) and
[`Points3D`](https://www.rerun.io/docs/reference/types/archetypes/points3d) archetypes, respectively. 
```

**Pull request merged**
